### PR TITLE
WT-7565 Update invalid backup configurations

### PR DESCRIPTION
**Pull request by etienneptl**

### CODE REVIEW

**Comment by etienneptl** [comment]
Sorry, WT-7541 went in, my bad. Reverted it. It is no longer part of this change as it's unrelated.

**Review by jiechenbo — APPROVED** [review]
LGTM

File: bench/wtperf/runners/multi-btree-backup.wtperf [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# wtperf options file: small btree multi-database configuration
+# Original cache was 500MB.  Shared cache is 500MB * database_count.
+conn_config="shared_cache=(name=pool,size=2500MB,chunk=1M),log=(enabled=false)"
+#conn_config="cache_size=250MB,log=(enabled=false)"
```
Comment (etienneptl) [inline bench/wtperf/runners/multi-btree-backup.wtperf:4]: Should we remove that line ?

Comment (jiechenbo) [reply bench/wtperf/runners/multi-btree-backup.wtperf:4]: I think we should be good with keeping this.

Comment (kommiharibabu) [reply bench/wtperf/runners/multi-btree-backup.wtperf:4]: The commented line is not required. Please remove that line from the original `multi-btree.wtperf` config also.

**Review by kommiharibabu — APPROVED** [review]
LGTM

**Pull request merged**